src/Components/article-content.jsx

```diff
@@ -0,0 +1,53 @@
+            <br/>
+            <br/>
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Eligendi deleniti ad unde quo id recusandae animi impedit perspiciatis ut, voluptas facilis itaque mollitia esse tempora! Commodi atque, autem doloribus quos libero dolorum consequuntur laudantium accusamus voluptates odit. Illo porro quos possimus harum iusto excepturi atque mollitia consectetur error nihil accusamus, voluptatum tempore labore pariatur eveniet impedit. Odit magni sit ut labore ullam unde ratione iusto nemo esse aperiam numquam odio repudiandae accusantium maxime quod aspernatur voluptatem iure voluptas quam quibusdam vitae, quaerat quos eveniet. Nostrum temporibus itaque voluptas id necessitatibus fuga non rem consectetur deserunt! Fuga dolorem sed rem in sit, quasi aliquam natus! Sequi, quasi possimus. Soluta aspernatur dolorum, ducimus totam incidunt ad, beatae recusandae repudiandae et rem dolores cum modi sit ratione. Fugiat voluptatum porro laborum rerum nobis accusantium tempora mollitia magnam blanditiis exercitationem, quae nulla alias nam asperiores voluptas, a, non omnis rem. Aut sit neque at excepturi. Dolores, a. Vitae in provident, excepturi a minus officiis debitis quo nihil voluptatibus cupiditate necessitatibus labore velit eveniet? Repudiandae tempore cum, molestiae asperiores nostrum eius quae voluptatibus saepe ad vero. Accusamus minus fugiat eveniet ratione deserunt nisi quam culpa veniam soluta alias, cum ipsam quas nemo ut dolorem illum!
+            <br/>
+            <br/>
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Perferendis dolores quisquam numquam odio quod nisi ut voluptates odit, repudiandae facere distinctio ullam! Officiis, delectus eveniet cupiditate ipsum itaque architecto laboriosam.
```

Comment:
You should be using the actual article body text that we get from the api here instead of hardcoded lorem ipsum